doc/02-Installation.md

```diff
@@ -12,6 +12,26 @@
 
 The module ships with database functions for calculating the host and service availability in `etc/schema/`.
 
+### Grant Required Privileges
+
+Skip this step if you used the database configuration wizard during the icinga2 installation.
```

Comment:
Please use `Icinga 2` instead of `icinga2` here.

Reply:
Done.